addons/web_kanban/static/src/js/kanban_view.js

```diff
@@ -91,14 +91,15 @@ var KanbanView = View.extend({
 
     willStart: function() {
         // add qweb templates
-        for (var i=0, ii=this.fields_view.arch.children.length; i < ii; i++) {
-            var child = this.fields_view.arch.children[i];
+        var fvg = $.extend(true, {}, this.fields_view);
+        for (var i=0, ii=fvg.arch.children.length; i < ii; i++) {
+            var child = fvg.arch.children[i];
```

Comment:
do not pass deep copy of fields_view to every place. it's only require here(`child = fvg.arch.children[i]`) because `transform_qweb_template` change fields_view object.